static/css/style.css

```diff
@@ -132,16 +152,17 @@ strong{
 }
 
 #frontpage{
-	height: 300px;
+	height: 400px;
 	padding: 5px 5px 20px 5px;
+	background: url(groupmeet.jpg) no-repeat fixed;
```

Comment:
are you sure you're running her branch? can you manually check if the image exists on your computer